source/features/comments-time-machine-links.tsx

```diff
@@ -32,45 +31,6 @@ async function updateURLtoDatedSha(url: GitHubURL, date: string): Promise<void>
 	select('a.rgh-link-date')!.pathname = url.assign({branch: oid}).pathname;
 }
 
-function addInlineLinks(menu: HTMLElement, timestamp: string): void {
-	const comment = menu.closest('.js-comment')!;
-
-	const links = select.all<HTMLAnchorElement>(`
-		[href^="${location.origin}"][href*="/blob/"]:not(.${linkifiedURLClass}),
-		[href^="${location.origin}"][href*="/tree/"]:not(.${linkifiedURLClass})
-	`, comment);
-
-	for (const link of links) {
-		const linkParts = link.pathname.split('/');
-		// Skip permalinks
-		if (/^[\da-f]{40}$/.test(linkParts[4])) {
-			continue;
-		}
-
-		const searchParameters = new URLSearchParams(link.search);
-		searchParameters.set('rgh-link-date', timestamp);
-		link.search = String(searchParameters);
-	}
-}
-
-function addDropdownLink(menu: HTMLElement, timestamp: string): void {
-	const dropdown = select('.show-more-popover', menu.parentElement!)!;
-
-	dropdown.append(
-		<>
-			<div className="dropdown-divider"/>
-			<a
-				href={buildRepoURL(`tree/HEAD@{${timestamp}}`)}
-				className={'dropdown-item btn-link ' + linkifiedURLClass}
-				role="menuitem"
-				title="Browse repository like it appeared on this day"
-			>
-				View repo at this time
-			</a>
-		</>,
-	);
-}
-
```

Comment:
I just moved those two functions closer to the `init` and shortened them a bit, but otherwise they haven't been modified.